config/schema.error.yaml

```
@@ -0,0 +1,575 @@
+    step: 'install-update: update'
+    title: 'Command `/bin/magento setup:upgrade` failed'
+    suggestion: 'Check the `cloud.log` and `install_upgrade.log` for more information. Add `VERBOSE_COMMANDS: ''-vvv''` into `.magento.env.yaml` for more detailed command output.'
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_CACHE_FLUSH_COMMAND_FAILED:
+    step: clean-cache
+    title: 'Command `/bin/magento cache:flush` failed'
+    suggestion: 'Check the `cloud.log` for more information. Add `VERBOSE_COMMANDS: ''-vvv''` into `.magento.env.yaml` for more detailed command output.'
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_MAINTENANCE_MODE_DISABLING_FAILED:
+    step: disable-maintenance-mode
+    title: 'Command `/bin/magento maintenance:disable` failed'
+    suggestion: 'Check the `cloud.log` for more information. Add `VERBOSE_COMMANDS: ''-vvv''` into `.magento.env.yaml` for more detailed command output.'
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_UNABLE_TO_READ_RESET_PASSWORD_TMPL:
+    step: 'install-update: reset-password'
+    title: 'Enable to read reset password template'
+    suggestion: ''
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_CACHE_ENABLE_FAILED:
+    step: 'install-update: cache_type'
+    title: 'Command `php ./bin/magento cache:enable` failed.'
+    suggestion: 'Command `php ./bin/magento cache:enable` runs only when Magento was installed but `./app/etc/env.php` file was absent or empty at the beginning of the deployment. Check the `cloud.log` for more information. Add `VERBOSE_COMMANDS: ''-vvv''` into `.magento.env.yaml` for more detailed command output.'
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_CRYPT_KEY_IS_ABSENT:
+    step: install-update
+    title: 'The `crypt/key`  key value does not exist in the `./app/etc/env.php` file or the `CRYPT_KEY` cloud environment variable'
+    suggestion: "This error occurs if the `./app/etc/env.php` file is not present when Magento deployment begins, or if the `crypt/key` value is undefined.\nIf you migrated the database from another environment, retrieve the crypt key value from that environment and add the value to the [CRYPT_KEY](https://devdocs.magento.com/cloud/env/variables-deploy.html#crypt_key) cloud environment variable in your current environment, see [Add the Magento encryption key](https://devdocs.magento.com/cloud/setup/first-time-setup-import-import.html#encryption-key).\nIf you accidentally removed the `./app/etc/env.php` file, use the following command to restore it from the backup files created from a previous deployment: `./vendor/bin/ece-tools backup:restore` CLI command .\""
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_ES_CANNOT_CONNECT:
+    title: 'Can''t get version of elasticsearch service'
```

```suggestion
    title: 'Can not connect to the Elasticsearch service'
```

I think Elasticsearch should be initial cap in text references for consistency (Elasticsearch vs. elasticsearch) -- unless you are referring to the Github repo or package name: https://github.com/elastic/elasticsearch